server/src/main/java/org/apache/seata/server/cluster/raft/RaftTransactionServerManager.java

```diff
@@ -0,0 +1,146 @@
+
+/**
+ */
+public class RaftTransactionServerManager extends AbstractRaftServerManager {
+
+    public static volatile RaftTransactionServerManager raftTransactionServerManager;
+
+    public static RaftTransactionServerManager getInstance() {
+        if (raftTransactionServerManager == null) {
+            synchronized (RaftTransactionServerManager.class) {
+                if (raftTransactionServerManager == null) {
+                    raftTransactionServerManager = new RaftTransactionServerManager();
+                }
+            }
+        }
+        return raftTransactionServerManager;
+    }
+
+    public void start() {
+        RAFT_SERVER_MAP.forEach((group, raftServer) -> {
+            try {
+                raftServer.start();
+            } catch (IOException e) {
+                logger.error("start seata server raft cluster error, group: {} ", group, e);
+                throw new RuntimeException(e);
+            }
+            logger.info("started seata server raft cluster, group: {} ", group);
+        });
+        if (rpcServer != null) {
+            rpcServer.registerProcessor(new PutNodeInfoRequestProcessor());
+            SerializerManager.addSerializer(SerializerType.JACKSON.getCode(), new JacksonBoltSerializer());
+            if (!rpcServer.init(null)) {
+                throw new RuntimeException("start raft node fail!");
+            }
+        }
+    }
+
+    public RaftTransactionServer getRaftServer(String group) {
+        return (RaftTransactionServer) RAFT_SERVER_MAP.get(group);
+    }
+
+    public Collection<RaftServer> getRaftServers() {
+        return RAFT_SERVER_MAP.values();
+    }
+
+    public static Set<String> groups() {
+        return RAFT_SERVER_MAP.keySet();
+    }
+
+    @Override
+    public void init() {
+        String dataPath;
+        String controllerInitConf = CONFIG.getConfig(ConfigurationKeys.SERVER_RAFT_CONTROLLER_SERVER_ADDR);
+        String initConf;
+        if (StringUtils.isBlank(controllerInitConf)) {
+            // It is currently in single-raft mode.
+            String group = CONFIG.getConfig(ConfigurationKeys.SERVER_RAFT_GROUP, DEFAULT_SEATA_GROUP);
+            dataPath = CONFIG.getConfig(ConfigurationKeys.STORE_FILE_DIR, DEFAULT_SESSION_STORE_FILE_DIR) + separator
+                    + "raft" + separator + serverId.getPort();
+            initConf = CONFIG.getConfig(ConfigurationKeys.SERVER_RAFT_SERVER_ADDR);
+            init(initConf);
+            Configuration configuration = new Configuration();
+            configuration.parse(initConf);
+            try {
+                RAFT_SERVER_MAP.put(group, createRaftServer(dataPath, group, configuration));
+            } catch (IOException e) {
+                throw new IllegalArgumentException("fail init raft cluster:" + e.getMessage(), e);
+            }
+        } else {
+            // todo: This needs to be modified to obtain the group and address list through TXG.
```

Comment:
At this point, make a request to the CG interface to retrieve all TXG information associated with the current IP, and then construct the Raft group.